TypeCobol/Compiler/Sql/CodeElements/SqlCodeElementBuilder.cs

```diff
@@ -474,5 +476,149 @@ private SymbolReference CreateTableOrAliasName(CodeElementsParser.TableOrAliasNa
+                        c => ParseTreeUtils.GetFirstToken(alterSequenceClauseContext.incrementClause()));
+                    if (alterSequenceClauseContext.incrementClause().numeric_constant != null)
+                    {
+                        incrementValue =
+                            new SqlConstant((Token) alterSequenceClauseContext.incrementClause().numeric_constant);
+                    }
+                }
+
+                if (alterSequenceClauseContext.minValueClause() != null)
+                {
+                    SetOption(alterSequenceClauseContext.minValueClause(), AlterSequenceClauseType.MinValue,
+                        c => ParseTreeUtils.GetFirstToken(alterSequenceClauseContext.minValueClause()));
+                    if (alterSequenceClauseContext.minValueClause().numeric_constant != null)
+                    {
+                        minValue = new SqlConstant((Token) alterSequenceClauseContext.minValueClause()
+                            .numeric_constant);
+                    }
+                }
+
+                if (alterSequenceClauseContext.maxValueClause() != null)
+                {
+                    SetOption(alterSequenceClauseContext.maxValueClause(), AlterSequenceClauseType.MaxValue,
+                        c => ParseTreeUtils.GetFirstToken(alterSequenceClauseContext.maxValueClause()));
+                    if (alterSequenceClauseContext.maxValueClause().numeric_constant != null)
+                    {
+                        maxValue = new SqlConstant((Token) alterSequenceClauseContext.maxValueClause()
+                            .numeric_constant);
+                    }
+                }
+
+                if (alterSequenceClauseContext.cycle() != null)
+                {
+                    SetOption(alterSequenceClauseContext.cycle(), AlterSequenceClauseType.Cycle,
+                        c => cycle = new SyntaxProperty<bool>(true, ParseTreeUtils.GetFirstToken(c)));
+                }
+
+                if (alterSequenceClauseContext.cacheClause() != null)
+                {
+                    SetOption(alterSequenceClauseContext.cacheClause(), AlterSequenceClauseType.Cache,
+                        c => ParseTreeUtils.GetFirstToken(c));
+                    if (alterSequenceClauseContext.cacheClause().numeric_constant != null)
+                    {
+                        cacheSize = new SqlConstant((Token) alterSequenceClauseContext.cacheClause().numeric_constant);
+                    }
+                }
+
+                if (alterSequenceClauseContext.SQL_ORDER() != null)
+                {
+                    SetOption(alterSequenceClauseContext.SQL_ORDER(), AlterSequenceClauseType.Ordered,
+                        c => ordered = new SyntaxProperty<bool>(true, ParseTreeUtils.GetFirstToken(c)));
+                }
+
+                if (alterSequenceClauseContext.noClauses() != null)
```

Comment:
NO MINVALUE and NO MAXVALUE are unhandled. You still have to call SetOption so as to add the clause in the set, just use an empty action

```csharp
SetOption(noKeyword, AlterSequenceClauseType.MinValue, c => {});
```